# This message holds status of all AR600 robot power supplies:
#  - 48v
#  - 12v
#  - 8v (2 supplies)
#  - 6v (2 supplies)

supply_state S48
supply_state S12
supply_state S8_1
supply_state S8_2
supply_state S6_1
supply_state S6_2

